app/Commands/InitializeCommand.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Commands;
+
+use App\Tools\Composer;
+use Illuminate\Support\Facades\Storage;
+use Laravel\Prompts\Concerns\Colors;
+use LaravelZero\Framework\Commands\Command;
+
+use function Laravel\Prompts\note;
+use function Termwind\terminal;
+
+class InitializeCommand extends Command
+{
+    use Colors;
+
+    protected $signature = 'initialize';
+
+    protected $description = 'Run Mise initialization steps and remove Mise (if defined in ./.mise/initialize.php)';
+
+    public function handle(): int
+    {
+        $miseDirectory = '.mise';
+
+        if (! $this->runScript("{$miseDirectory}/initialize.php")) {
+            return 1;
+        }
+
+        if (Storage::deleteDirectory($miseDirectory)) {
+            $this->mise("Deleted {$this->heavy($miseDirectory)} directory");
+        } else {
+            $this->miseError("Failed deleting {$this->heavy($miseDirectory)}; please delete manually.");
+        }
+
+        $package = 'tightenco/mise';
+        $composer = app(Composer::class);
```

Comment:
Can you do dependency injection in the constructor or `handle` method of a command?

Reply:
Apparently, you can in the handle method.